plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -552,6 +552,12 @@ private function add_inventory_group_blocks() {
 					'label'    => __( 'Track stock quantity for this product', 'woocommerce' ),
 					'property' => 'manage_stock',
 					'disabled' => 'yes' !== get_option( 'woocommerce_manage_stock' ),
+					'disabledCopy' => sprintf(
+						/* translators: %1$s: Learn more link opening tag. %2$s: Learn more link closing tag.*/
+							__( 'Per your %1$sstore settings%2$s, inventory management is <strong>disabled</strong>.', 'woocommerce' ),
+							'<a href="http://blue.local/wp-admin/admin.php?page=wc-settings&tab=products&section=inventory" target="_blank" rel="noreferrer">',
```

Comment:
Caught!!!! We should not have a static global url here. Please use the function util we have to generate the woo admin url.

Reply:
Good call! A classic copy 🍝
I fixed the URL in the commit [f159bd1](https://github.com/woocommerce/woocommerce/pull/40011/commits/f159bd1639935b4cf4abf40984f18e4ee32678e1)